docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
+  }
+}
+```
+
+Sample response:
+```json
+{
+  "inference_results": [
+    {
+      "output": [
+        {
+          "name": "response",
+          "dataAsMap": {
+            "response": [
+              "Platform",
+              "Search for anything",
+              "Platform",
+              "Search",
+              "for",
+              "anything"
+            ]
+          }
+        }
+      ],
+      "status_code": 200
+    }
+  ]
+}
+``` 
+
+### Step 6: Create an ingest pipeline with an ml_inference processor
+
+- The ML inference processor passes a source field to a model, extracts the model output, and inserts it into a target field.
+
+- This example processor works as follows:
+
+  - [1] Extracts values from the image field and passes the values to the image_bytes parameter.
+  - [2] Invokes the Amazon Rekognition DetectText API providing the image_bytes parameter.
+  - [3] Extracts values from the DetectText API response with JSON path.
+  - [4] Inserts the extracted values into the text field.
+  - [5] Removes the original image field.
```

Comment:
`Removes original image field` may confuse users to think ML Inference Processor can do that. Maybe refactoring to 

# Step 6: Create Ingest pipeline
Explanation that the pipeline has two processors:
- ML Inference processor:
  -  Extracts values from the image field and passes the values to the image_bytes parameter.
  -  Invokes the Amazon Rekognition DetectText API providing the image_bytes parameter.
  -  Extracts values from the DetectText API response with JSON path.
  -  Inserts the extracted values from the API into a new field within the same document.
- Remove processor:
   -  Removes the base64 string field in the original document for clarity

For an idea